README.md

```diff
@@ -1,2 +1,21 @@
 # sd-jwt
+
+## Description
+
 Library for the production, verification and consumption of SD-JWT documents
+
+## Installation
+
+### Maven
+
+Add the following dependency to your `pom.xml` file:
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>de.adorsys</groupId>
+    <artifactId>sd-jwt</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
Is the SNAPSHOT required ?

Reply:
I suggest to add a placeholder encouraging consumers to look for for the latest version on Maven.